python/oneflow/nn/graph/graph.py

```diff
@@ -1342,6 +1347,16 @@ def __del__(self):
             return
         oneflow._oneflow_internal.eager.Sync()
 
+    def __ensure_input_tensors_contiguous(self, *args, **kwargs):
+        io_node = IONode(None, 0, (args, kwargs), "_" + self.name + "_" + "input")
+
+        def leaf_node_fn(node):
+            if isinstance(node._value, Tensor) and not node._value.is_contiguous():
+                node._value.contiguous_()
+            return node
+
+        io_node.map_leaf(leaf_node_fn)
+
```

Comment:
Need to add  a test of this case:
- input is not contigous
- parameter is not contigous